### PR TITLE
Cataclysm Poem grammatical error fix

### PR DESCRIPTION
**Pull request by SomeGuyIGues**
#### Summary

Bugfixes "Fixes a grammatical error within the cataclysm poem. It is now perfect."

#### Purpose of change

Continuation of #74123. It is complete.

#### Describe the solution

It's not its. 

#### Describe alternatives you've considered

Submit to Kevin's mind control and not fix the error.

#### Testing

I've read that sentence way too many times, I know it's correct. Its? No it's... right?

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -4,7 +4,7 @@
  * Who knows
  */
 
-// KG: Yes the above is inaccurate now, it's also a poem, it's stays.
+// KG: Yes, the above is inaccurate now. It's also a poem, its stays.
```
Comment (Karol1223) [inline src/main.cpp:7]:
```suggestion
// KG: Yes, the above is inaccurate now. It's also a poem, it stays.
```

**Review by Karol1223 — CHANGES_REQUESTED** [review]

**Comment by Maleclypse** [comment]
@kevingranade

**Review by ashGlaw — APPROVED** [review]
Grammar fixed 👍

**Comment by Maleclypse** [comment]
> @kevingranade
> 
> edit: to the people responding via emoji. I am summoning kevin in case his poem requires a specific syllable structure that grammatical fixes might break.

Oh I’m stupid you aren’t fixing grammar in the poem just in the comment as to why it stays.

**Pull request merged**